flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/BaseReconciler.java

```diff
@@ -60,7 +59,11 @@ private DeleteControl shutdownAndDelete(
             FlinkUtils.deleteCluster(flinkApp, kubernetesClient, true);
         }
         IngressUtils.updateIngressRules(
-                flinkApp, effectiveConfig, operatorNamespace, kubernetesClient, true);
+                flinkApp,
+                effectiveConfig,
+                flinkApp.getMetadata().getNamespace(),
```

Comment:
this do not have pass the `flinkApp.getMetadata().getNamespace()`, it can get from the `flinkApp`

Reply:
yup, I missed that. Thanks!